frontend update finished

BananeGuimauve opened the pull request.

Commit:
frontend update in progress

Commit:
Frontend update finished. The frontend was generated using JHipster 6.0.1.

danielpetisme commented on src/JHipsterNetSampleApplication/ClientApp/WEB-INF/web.xml:
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app
```
I'm not sure .Net need a Java `web.xml` file 😄

danielpetisme reviewed (commented):
Can you tell me by comment the files you actually updated for the .Net backend needs. please.

BananeGuimauve commented:
To summerize : 
- I copied pasted the new ClientApp
- Wherever there was the keyword "jhipsterSample" I changed to "JhipsterNetSample". Wherever there was the keyword "java" I changed to ".Net"
- I updated by comparison to their new versions the files "webpack.common.js", "webpack.dev.js", "webpack.prod.js", "webpack.test.js" and "angular.json" (no copy paste)
- I added the missing packages and updated the versions numbers of the existing packages in the package.json file
- I will push now the files "tsconfig.json", "tsconfig-aot.json", "tslint.json" that I forgot to update but they have minor changes

All the files I mentionned are either in the "ClientApp" directory or in the "webpack" directory or at the root of the JHipsterNetSampleApplication project.

Commit:
Delete unnecessary file and add missing modified files
